application/org.openjdk.jmc.flightrecorder.graphview/src/main/java/org/openjdk/jmc/flightrecorder/graphview/views/GraphView.java

```diff
@@ -86,14 +86,14 @@
 public class GraphView extends ViewPart implements ISelectionListener {
 	private static final String HTML_PAGE;
 	static {
-		String jsD3V5 = "jslibs/d3.v5.min.js";
+		String jsD3 = "jslibs/d3.v6.min.js";
```

Comment:
I wonder if we can rename the js lib to avoid code modification. And only update the pom when changing version.

Reply:
That's a good idea! Feel free to open a separate PR for this.